src/models/signer/config.rs

```diff
@@ -46,6 +46,16 @@ pub struct TurnkeySignerFileConfig {
     pub public_key: String,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+#[serde(deny_unknown_fields)]
+pub struct CdpSignerFileConfig {
+    pub api_key_id: String,
+    pub api_key_secret: PlainOrEnvValue,
+    pub wallet_secret: PlainOrEnvValue,
+    pub evm_account_address: Option<String>,
+    pub solana_account_address: Option<String>,
```

Comment:
Lets add single  required `address` field that works for all networks rather than network specific fields. As we add more networks the latter will lead to field sprawl and a single field is consistent with our other signers.

I would validate this field in `validate_cdp_config()` that it matches either evm/solana format. Perhaps we can add some util funcs for matching. One idea here might be to parse the addresses using solana and alloy sdk